tensorflow/contrib/learn/python/learn/datasets/mnist.py

```diff
@@ -223,23 +223,25 @@ def fake():
   TEST_IMAGES = 't10k-images-idx3-ubyte.gz'
   TEST_LABELS = 't10k-labels-idx1-ubyte.gz'
 
-  local_file = base.maybe_download(TRAIN_IMAGES, train_dir,
-                                   SOURCE_URL + TRAIN_IMAGES)
+  def download_local_file(fname):
+    try:
+      return base.maybe_download(fname, train_dir, SOURCE_URL + fname)
```

Comment:
I guess if we're expanding to a list of backup URLs, I feel like this change would require significant changes to other files I haven't looked at!